packages/larva-patterns/modules/content-showcase-grid/content-showcase-grid.twig

```diff
@@ -0,0 +1,7 @@
+<div class="content-showcase-grid larva // {{ content_showcase_classes }}">
+	<div class="{{ content_showcase_items_classes }}">
```

Comment:
add `content-showcase-grid-items` to this element.

Reply:
@laras126  This has been fixed in the updated template